translations/hr.ts

```diff
@@ -1903,7 +1903,7 @@ Dijelite ih sa svojim prijateljima kako biste komunicirali.</translation>
     <message>
         <source>Ignoring non-Tox file</source>
         <comment>popup title</comment>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Ignoriram ne-Tox datoteku</translation>
```

Comment:
@zetok. Stop. We have some problems. I doesn't write this. It seens like a problem with Weblate
![spectacle lh1461](https://cloud.githubusercontent.com/assets/6306608/15100787/e80a5658-158e-11e6-905d-13d061548a97.png)